contracts/UniswapV3Staker.sol

```diff
@@ -193,14 +194,12 @@ contract UniswapV3Staker is ERC721Holder, ReentrancyGuard {
     function depositToken(uint256 tokenId) external {
         // TODO: Make sure the transfer succeeds and is a uniswap erc721
         // I think this is not secure
-        nonfungiblePositionManager.safeTransferFrom(
+        nonfungiblePositionManager.transferFrom(
```

Comment:
`safeTransferFrom` would call into the `onERC721ReceivedHook` and deposit twice.